app/components/marketplace-plugin-list/constants.js

```diff
@@ -35,5 +35,40 @@ $ appknox upload &lt;path to apk/ipa file&gt; | xargs appknox cicheck --risk-thr
 </pre>
       </li>
     </ul>
-  `
+  `,
+appCenterInstructions:`
+  <ul class="bullet-list margin-r-1 margin-l-1h">
+    <li class="margin-b-1">
+      Generate a personal access token from <a href="/settings/developersettings" target="_blank">Developer Settings</a>
+    </li>
+    <li class="margin-b-1">
+      Create a file <code class="black-text">appcenter-post-build.sh</code> in your source code repository and add the content given below:
+      <pre class="code-block">
+if [ "$AGENT_JOBSTATUS" == "Succeeded" ]; then
+  if [ "$APPCENTER_BRANCH" == "master" ];
+    then
+        curl -L https://github.com/appknox/appknox-go/releases/download/1.1.0/appknox-\`uname -s\`-x86_64 > appknox && chmod +x appknox
+        ./appknox upload $APPCENTER_OUTPUT_DIRECTORY/app-release.apk
+        rm appknox
+  else
+      echo "Current branch is $APPCENTER_BRANCH"
+  fi
+fi</pre>
+      <small>(More info: <a href="https://aka.ms/docs/build/custom/scripts" target="_blank">https://aka.ms/docs/build/custom/scripts)</a></small>
+      <p class="margin-t-1 alert-info padding-10">
+          Make sure you use the latest version of the appknox binary in the above curl URL
```

Comment:
We are not mentioning this warning in other instructions so we can skip it here also.

As we will specifically be informing our customers whenever major releases happens.

Reply:
Done. Removed it.